challenges/FizzBuzzTree/fizz_buzz_tree.py

```diff
@@ -0,0 +1,35 @@
+def fizz_buzz(node):
+    """Helper function for fizzbuzz conditionals."""
+    if node.val % 5 == 0 and node.val % 3 == 0:
+        node.val = 'FizzBuzz'
+        return node.val
+    elif node.val % 3 == 0:
+        node.val = 'Fizz'
+        return node.val
+    elif node.val % 5 == 0:
+        node.val = 'Buzz'
+        return node.val
+    else:
+        return node.val
+
+
+def fizz_buzz_tree(tree):
+    """Traverses a tree and executes fizzbuzz helper at each node."""
+    temp = []
+    if tree.root is None:
+        raise ValueError('This tree is empty!')
```

Comment:
You do not need to raise an exception here since this is basically just a traversal of the tree. If the tree passed in is empty, then the tree returned will be empty.